specification/signalr/resource-manager/Microsoft.SignalRService/preview/2018-03-01-preview/signalr.json

```diff
@@ -22,7 +22,7 @@
           "SignalR"
         ],
         "description": "Lists all of the available REST API operations of the Microsoft.SignalRService provider.",
-        "operationId": "Signalr_ListOperations",
+        "operationId": "SignalR_ListOperations",
```

Comment:
You should call this one `Operations_List`. It will be consistent with everybody else:
https://github.com/Azure/azure-rest-api-specs/blob/9666a766d478b21b587210260fb7de342ae9980e/specification/storage/resource-manager/Microsoft.Storage/stable/2017-10-01/storage.json#L29

Reply:
Issue #2899

Reply:
updated